samples/CuriePME.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2017, Intel Corporation.
+
+// Test code to use the Curie Pattern Matching Engine within the Intel Curie Compute
+// on the Arduino 101/tinyTile, which uses the CuriePME API to train and classify data
+console.log("Curie Pattern Matching Engine test...");
+
+// import pme module
+var CuriePME = require("pme");
+
+var training1 = [ 10, 10, 10, 10 ];
+var training2 = [ 20, 20, 20, 20 ];
+
+var classify1 = [ 15, 15, 15, 15 ];
+var classify2 = [ 14, 14, 14, 14 ];
+var classify3 = [ 16, 16, 16, 16 ];
+var classify4 = [ 25, 25, 25, 25 ];
+var classify5 = [ 30, 30, 30, 30 ];
+
+CuriePME.begin();
+
+var mode = CuriePME.getClassifierMode();
```

Comment:
@poussa @zolkis, any preferences here?  I am fine with either, since it's our own API.